ui/components/ui/tabs/tabs.types.ts

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { BoxProps } from '../../component-library/box/box.types';
+import { TextProps } from '../../component-library/text/text.types';
+
+export type TabProps<TKey extends string = string> = {
```

Comment:
TKey is a generic string type so it can allow for `AccountOverviewTabKey` which is an enum with string values